privatelink/gcp/terraform/privatelink.tf

```
@@ -0,0 +1,138 @@
+terraform {
+  required_version = ">= 0.12.17"
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "4.11.0"
+    }
+    external = {
+      source  = "hashicorp/external"
+      version = "2.0.0"
+    }
+    time = {
+      source  = "hashicorp/time"
+      version = "0.6.0"
+    }
+  }
+}
+
+provider "google" {
+  project = var.realm
+  region  = var.region
+}
+
+variable "realm" {
```

```suggestion
variable "project" {
```

Not useful to leak our internal cloud agnostic terminology to customer experience.